src/index.html

```diff
@@ -11,7 +11,172 @@
+            name="Name"
+            required
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="Age"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="date"
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="checkBox"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="12"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cat"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="cat"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+            step="2"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="cars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="box">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="info"
+            cols="20"
+            rows="2"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="info"
            cols="20"
            rows="2"
            autocomplete="off"
        ></textarea>
```
It will fix this problem
![image](https://user-images.githubusercontent.com/96409563/221013849-5bb83aa3-45cd-45ba-a4f6-cfd2b86c176f.png)
![image](https://user-images.githubusercontent.com/96409563/221013947-aebfc8a3-298e-470d-91fc-bbb737792976.png)